rhizome.js

```diff
@@ -252,15 +267,16 @@ function highlightNode(target_id) {
     }
     idNodeOpacity(d.source.id);
     idNodeOpacity(d.target.id);
+    idLinkColor(d.source.id+"-"+d.target.id);
   });
 
   second_degree_nodes.forEach(function(n){
-    console.log('node',n);
     links.filter(function(d) {
       return ((d.source.id == n) || (d.target.id == n));
     }).forEach(function(d){
       idNodeOpacity(d.source.id);
       idNodeOpacity(d.target.id);
+      idLinkColor(d.source.id+"-"+d.target.id);
```

Comment:
second-order links could be a different colour...?

Reply:
Yes!